models/cow_protocol/gnosis/cow_protocol_gnosis_solvers.sql

```diff
@@ -1,4 +1,6 @@
-{{ config(alias='solvers',
+{{ config(
+        alias='solvers',
```

Comment:
Does this need an `alias()` wrapper around it?

Reply:
```suggestion
        alias=alias('solvers'),
```